### DISCUSSION TITLE
Non-Openai clients for autogen 0.4

### DISCUSSION THREAD

**Discussion by rusenbb** [microsoft/autogen#3854] [Q&A]
Will there be Antrophic (and other clients as well) clients to use on autogen 0.4? I could not figure out a way to try different models. And in the examples only Openai is used.

**Comment by ekzhu** [comment]
Yes. It is on our roadmap to port the clients in 0.2 forward. Would you be interested in creating a community module? 

See: https://microsoft.github.io/autogen/dev/user-guide/extensions-user-guide/index.html#creating-your-own-community-extension
